ui/app/components/evaluation-sidebar/detail.js

```diff
@@ -75,12 +75,4 @@ export default class Detail extends Component {
   closeSidebar() {
     return this.statechart.send('MODAL_CLOSE');
   }
-
-  keyCommands = [
-    {
-      label: 'Close Evaluations Sidebar',
-      pattern: ['Escape'],
-      action: () => this.closeSidebar(),
-    },
-  ];
```

Comment:
`question` What's this doing here? Do we want to remove this behavior?

Reply:
Ack, great catch! I did not mean to remove this; probably cut when I meant to copy.